### PR TITLE
Implementing the Ballerina Shopify Connector

### LINKED ISSUE #1: Implement Shopify Connector
**Description:**

- Implement Shopify Connector with the following features:
  - StoreClient
  - CustomerClient
  - ProductClient
  - OrderClient

  These clients will provide the CRUD operations.

### PR DESCRIPTION
**Pull request by ThisaruGuruge**
## Purpose
Implement the Ballerina Shopify Connector.

This connector enables to connect with Shopify REST API to handle Shopify Customers, Products, and Orders.

Compatible with Ballerina 1.2.x releases.

1. [Design Document](https://docs.google.com/document/d/1O6PfSKXsug2NTk5K_oaXjMS9kXMrJ9ZZOYduv1C5NC4/edit#heading=h.fwh9mko8juex)

Fixes #1

### CODE REVIEW

File: src/shopify/tests/customer_test.bal [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+    if (result is Error) {
+        test:assertFail(result.toString());
+    }
+}
+
+@test:Config {}
+function deleteNonExistingCustomerTest() {
+    Error? result = customerClient->remove(9999);
+    if (result is Error) {
+        test:assertEquals(result.message(), "Not Found");
```
Comment (chamil321) [inline src/shopify/tests/customer_test.bal:170]: This change is not compatible with 1.2.x.

File: src/shopify/tests/product_test.bal [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+
+@test:Config {}
+function getAllProductsWithInvalidLimitTest() {
+    string expectedMessage = "The max limit must be a positive integer less than 250 (inclusive)";
+    ProductFilter filter = {
+        'limit: 500
+    };
+    var getAllResult = productClient->getAll(filter);
+    if (getAllResult is Error) {
+        string actualMessage = getAllResult.message();
```
Comment (chamil321) [inline src/shopify/tests/product_test.bal:266]: This API is not supported in 1.2.x

File: src/shopify/tests/customer_test.bal (hunk 1 above)
Comment (ThisaruGuruge) [reply src/shopify/tests/customer_test.bal:170]: This is not intended to be here. There's something wrong with this branch, it has `migrate to Ballerina Swan Lake Release` commit, and I don't have it locally. Force pushing doesn’t do anything. Will close this PR, and create new one.

**Comment by ThisaruGuruge** [comment]
Closing since this PR is sent from the wrong branch.

**Pull request closed without merging**